R-package/R/xgb.plot.importance.R

```diff
@@ -1,64 +1,75 @@
-#' Plot feature importance as a bar graph
+#' Plot feature importance
 #'
 #' Represents previously calculated feature importance as a bar graph.
-#' \code{xgb.plot.importance} uses base R graphics, while \code{xgb.ggplot.importance} uses the ggplot backend.
+#' - `xgb.plot.importance()` uses base R graphics, while
+#' - `xgb.ggplot.importance()` uses "ggplot".
```

Comment:
Is the double quote for ggplot an explicit choice? (instead of {ggplot}).

Reply:
Curly braces created problems in help files, so we can't use them.